frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/provision-source.tsx

```diff
@@ -114,4 +140,15 @@ type ProvisionSourceComponentProps = {
   onChange: (key: string, value: string | boolean) => void;
   goToStorageStep: () => void;
   goToNetworkingStep: () => void;
+  activeNamespace: string;
 };
+
+const mapStateToProps = (state: RootState): { activeNamespace: string } => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
```

Comment:
just a note for the future: we have our own variable in the redux which might be subjected to change

Reply:
using a function now